cmake/Modules/UrhoCommon.cmake

```diff
@@ -389,6 +389,9 @@ if (EMSCRIPTEN)
     set (URHO3D_LIB_TYPE STATIC)
     unset (URHO3D_LIB_TYPE CACHE)
 endif ()
+if (URHO3D_TRACY_PROFILING)
+    set (URHO3D_PROFILING 0)
```

Comment:
Nit-picking, but urho build system seems to prefer `TRUE`|`FALSE`. Consistency is a sign of quality. 👍🏻

Reply:
TRUE|FALSE and 1|0 are actually mixed.  Please see the lines before mine, they are using 1 and 0. Let's keep using 0, until someone refactor the scripts.

Reply:
I mean I'm not supposed to change those previous lines using `0` or `1`,  so I have to use 0 following those lines. We can have another PR to unify that.

Reply:
URHO3D_PROFILING is option I'm not sure if the options can be changed in this way, I will test it in the evening